source/nordic_sdk/components/ble/common/ble_srv_common.c

```diff
@@ -39,7 +39,7 @@
 #include <string.h>
 #include "nordic_common.h"
 #include "app_error.h"
-
+#include "ble.h"

```

Comment:
@jrobeson Yes it can be a problem on windows as well. So you should always use `#include ble/BLE.h` to use the BLE API.